getsolus/packages
# delta: Add at 0.17.0

**Summary**

Add delta - a syntax-highlighting pager for git, diff, and grep output.

- Resolves https://github.com/getsolus/packages/issues/2275

**Test plan**

All following scenarios should output neatly formatted and highlighted
diffs:

- Use `delta` as replacement for `diff`:
  ```console
  $ delta /usr/share/defaults/etc/profile /usr/share/defaults/etc/xprofile
  ```
- Use `delta` in git repository:
  ```console
  $ GIT_PAGER=delta git diff HEAD~1
  ```
- Use `delta` with side-by-side mode in git repository:
  ```
  $ GIT_PAGER=delta DELTA_FEATURES='+side-by-side' git diff HEAD~1
  ```

**Checklist**
